src/main/query_profiler.cpp

```diff
@@ -640,6 +656,22 @@ void QueryProfiler::WriteToFile(const char *path, string &info) const {
 	}
 }
 
+profiler_settings_t ErasePhaseTimingSettings(profiler_settings_t settings) {
```

Comment:
Does that mean they are still present in the `set`? Does using a different set for operators and the query makes sense?

Reply:
This function is used to erase the timings from the settings when the query tree is initialized. This means they never get stored in the metrics map for each operator and only in the query node.

Reply:
Amazing